ext/session/session.c

```diff
@@ -2385,7 +2387,6 @@ PHP_FUNCTION(session_create_id)
 	if (prefix && ZSTR_LEN(prefix)) {
 		if (php_session_valid_key(ZSTR_VAL(prefix)) == FAILURE) {
```

Comment:
The length of the prefix could already be checked here as a preliminary sanity check?

Reply:
A `ValueError` appears to be appropriate.

Reply:
I changed the logic and now it throws ValueError for a longer string.